core/build/default.go

```diff
@@ -1,8 +1,5 @@
-//go:build !dev
+//go:build !dev && !test
 
 package build
 
-// Dev is a build tag that allow use of insecure config family.
-// it's set to false by default, building the node with -tags=dev allows it to be overridden
-// and enable the use of insecure configs.
-const Dev = false
+const Mode = Prod
```

Comment:
If the helper methods are the preferred API, should we unexport this?
```suggestion
const mode = Prod
```